dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java

```diff
@@ -109,6 +109,6 @@ public static FieldSql getInstance() {
 	public String deleteFieldVarsForField= "delete from field_variable where field_id = ?";
 	public String insertFieldVar= 	"insert into field_variable( id, field_id, variable_name, variable_key, variable_value, user_id, last_mod_date) values (?,?,?,?,?,?,?)";
 
-	public static final String moveSorOrderForward = "update field set sort_order = sort_order + 1 where sort_order >= ? and sort_order < ?";
-	public static final String moveSorOrderBackward = "update field set sort_order = sort_order - 1 where sort_order > ? and sort_order <= ?";
+	public static final String moveSorOrderForward = "update field set sort_order = sort_order + 1 where structure_inode = ? and sort_order >= ? and sort_order < ?";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 162 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)